bin/lintsite

```diff
@@ -0,0 +1,111 @@
+
+const CMD_LINT_YAML = 'lint:yaml';
+const CMD_LINT_TWIG = 'lint:twig';
+
+date_default_timezone_set('UTC');
+set_time_limit(0);
+
+(@include_once __DIR__ . '/../vendor/autoload.php') || @include_once __DIR__ . '/../../../autoload.php';
+
+function initYamlLinter(Application $application) {
+    $application->add(new Symfony\Component\Yaml\Command\LintCommand());
+}
+
+function initTwigLinter(Application $application) {
+    $twigLintCommand = new Symfony\Bridge\Twig\Command\LintCommand();
+
+    $twigEnvironment = new \Twig_Environment(new \Twig_Loader_Filesystem());
+
+    $twigEnvironment->addExtension(new Twig_Extensions_Extension_Text());
+    $twigEnvironment->addExtension(new Twig_Extensions_Extension_I18n());
+    $twigEnvironment->addExtension(new Twig_Extensions_Extension_Intl());
+    $twigEnvironment->addExtension(new Twig_Extensions_Extension_Date());
+    $translator = new Symfony\Component\Translation\Translator('en_US');
+    $twigEnvironment->addExtension(new Symfony\Bridge\Twig\Extension\TranslationExtension($translator));
+
+    $twigEnvironment->addFunction(new Twig_SimpleFunction('asset', function () {}));
+    $twigEnvironment->addFunction(new Twig_SimpleFunction('image', function () {}));
+    $twigEnvironment->addFunction(new Twig_SimpleFunction('path', function () {}));
+
+    $twigLintCommand->setTwigEnvironment($twigEnvironment);
+    $application->add($twigLintCommand);
+}
+
+function getSitesDirs()
+{
+    $finder = Finder::create()->directories()->in(SITES_DIR)->depth('== 0');
+    return $finder->getIterator();
+}
+
+function buildSiteYmlFileList($siteDir)
+{
+    $list = [];
+    $finder = Finder::create()->files()->in(SITES_DIR . '/' . $siteDir)->name('*.yml')->depth('== 1');
+    $iterator = $finder->getIterator();
+    while ($iterator->valid()) {
+        $list[] = $iterator->current()->getFilename();
+        $iterator->next();
+    }
+
+    if (file_exists(SITES_DIR . '/' . $siteDir . '/' . DATA_DIR)) {
+        $list[] = SITES_DIR . '/' . $siteDir . '/' . DATA_DIR;
+    }
+
+    if (file_exists(SITES_DIR . '/' . $siteDir . '/' . LANGUAGES_DIR)) {
+        $list[] = SITES_DIR . '/' . $siteDir . '/' . LANGUAGES_DIR;
+    }
+
+    return $list;
+}
+
+function lintSiteFiles(Application $application, $siteDir)
+{
+    $output = new \Symfony\Component\Console\Output\ConsoleOutput();
+    foreach (buildSiteYmlFileList($siteDir) as $node) {
+        $application->run(new ArgvInput(['', 'lint:yaml', $node]));
+    }
+    $output->write(sprintf('<info>%s</info>', $siteDir));
+    $application->run(new ArgvInput(['', 'lint:twig', SITES_DIR . '/' . $siteDir . '/' . TEMPLATES_DIR]));
+}
+
+$application = new Application('TwigYard', '0.1.0');
+initYamlLinter($application);
+initTwigLinter($application);
+$application->setAutoExit(false);
+
+$argv = $_SERVER['argv'];
+$argc = $_SERVER['argc'];
+
+if ($argc === 2) {
+    lintSiteFiles($application, $argv[1]);
+} elseif ($argc === 1) {
```

Comment:
If no args or illegal args are submitted help should be shown.
If `lintsite --all` is run all sites should be checked.